includes/admin/upgrades/upgrade-functions.php

```diff
@@ -3593,3 +3597,30 @@ function give_v2511_upgrades() {
 		$wp_roles->remove_cap( $role, 'read_give_payment' );
 	}
 }
+
+/**
+ * Upgrade for version 2.6.3
+ *
+ * @since 2.6.3
+ */
+function give_v262_upgrades() {
```

Comment:
@ravinderk why is this function named `give_v262_upgrades()` when it's going into 2.6.3?

Reply:
@DevinWalker By mistake, I name like that. I will rename it.

Reply:
fixed: https://github.com/impress-org/givewp/compare/84cf543e756b...9f5b722ed764